packages/gatsby-theme-styleguide/src/gatsby-components/Menu/__tests__/__snapshots__/MenuVersionSelect.test.js.snap

```diff
@@ -25,7 +48,7 @@ exports[`<MenuVersionSelect /> render corectly 1`] = `
     <option
       value="default"
     >
-      -- choose --
+      Choose version
```

Comment:
that was the text pattern recommended by our own design system, but since we decided ourself that it was stupid to follow our own rules... I suppose we can do what the fuck we want. ;)

Reply:
fixed